public/index.html

```diff
@@ -75,38 +75,31 @@ <h4>Doctor's Details <span class="step">(Step 2 / 6)</span></h4>
 	            				<div class="form-group">
 				                    <label>Patient's Full Name:</label><br>
-				                    <input type="text" name="patient_name" class="form-control" id="patient_name" required>
-				                </div>
-	            				<div class="form-group">
-				                    <label>Age:</label><br>
-				                    <input type="number" name="patient_age" class="form-control" id="patient_age" required>
-				                </div>
-				                <div class="form-group">
-									<label>Gender:</label><br>
-					                <select class="form-control" id="patient_gender" name="patient_gender" required>
-					                	<option value="Male">Male</option>
-					                	<option value="Female">Female</option>
-					                	<option value="Other">Other</option>
-					                </select>
+				                    <input type="text" name="patient_name" placeholder="Full Name" class="form-control" id="patient_name" required>
 								</div>
-				                <div class="form-group">
-				                    <label>Location:</label><br>
-				                    <input type="text" name="patient_location" class="form-control" id="patient_location" required>
-				                </div>
-				                <div class="form-group">
-									<label>Blood Group:</label><br>
-					                <select class="form-control short" id="patient_blood_group" name="patient_blood_group" required>
-					                	<option value="AB+">AB+</option>
-										<option value="AB-">AB-</option>
-					                	<option value="A+">A+</option>
-					                	<option value="A-">A-</option>
-					                	<option value="B+">B+</option>
-					                	<option value="B-">B-</option>
-										<option value="O+">O+</option>
-					                	<option value="O-">O-</option>
-
-					                </select>
+								<div class="form-group">
+									<label>Patient Email Id:</label><br>
+					                <input type="text" name="doctor_email" placeholder="joe@example.com" class="form-control" id="doctor_email">
+								</div>
+								<div class="form-row">
+									<div class="form-group">
+										<label>Gender:</label><br>
+										<select class="form-control" id="patient_gender" name="patient_gender" required>
+											<option value="Male">Male</option>
+											<option value="Female">Female</option>
+											<option value="Other">Other</option>
+										</select>
+									</div>
+									<div class="form-group">
+										<label>Location:</label><br>
+										<input type="text" name="patient_location" placeholder="Mumbai, Kolkata" class="form-control" id="patient_location" required>
+									</div>
+									<div class="form-group">
+										<label>Age:</label><br>
+										<input type="number" name="patient_age" placeholder="Your Age" class="form-control" id="patient_age" required>
```

Comment:
This form will be filled by doctors. So, placeholder for patient's age cannot be 'Your age'

Reply:
Sorry for that silly mistake.